webapp/channels/src/components/post_edit_history/edited_post_item/edited_post_item.tsx

```diff
@@ -205,7 +211,11 @@ const EditedPostItem = ({post, isCurrent = false, postCurrentVersion, theme, act
         <CompassThemeProvider theme={theme}>
             <div
                 className={postContainerClass}
-                onClick={togglePost}
+                onClick={() => {
+                    if (!open) {
+                        setOpen(true);
+                    }
+                }}
```

Comment:
Making it so that clicking on the outermost div always expands the item seems a bit odd to me. I understand that you're doing this to make it so that you can select text easier which makes sense, but I feel like a better approach might be to make it so that only the inner `edit-post-history__title__container` div is clickable, but it should cover the whole space of the item when it's collapsed.

In this screenshot, the yellow/blue areas are this div when it's collapsed/expanded and the green area is the inner div. Could we perhaps expand the inner div to fill the margins around it so that you can click anywhere above it to toggle? If we do that, we won't need the click handler on this part because the inner div will cover the whole thing when it's collapsed
![Screenshot 2024-06-17 at 3 11 27 PM](https://github.com/mattermost/mattermost/assets/3277310/ebc8b323-bf5b-4add-a96f-10b1eb328029)